pokemons/helpers.py

```diff
@@ -28,9 +28,9 @@ def is_pokemons_sum_valid(pokemons):
     pokemons_args = [get_pokemon_args(pokemon) for pokemon in pokemons]
     stats = []
 
-    for i in range(0, 3):
+    for poke_arg in pokemons_args:
         for x in keys:
```

Comment:
As a good practice, avoid naming variables as random letters, always name them for what they mean, no matter how insignificant they are :)

Reply:
Done!